experiments/openebs/openebs-pool-network-delay/openebs_pool_network_delay_ansible_logic.yml

```diff
@@ -0,0 +1,126 @@
+- hosts: localhost
+  connection: local
+
+  vars:
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_pvc: "{{ lookup('env','APP_PVC') }}"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_experiment: "openebs-pool-network-delay"
+    c_interval: "{{ lookup('env','CHAOS_INTERVAL') }}"
+    data_persistence: "{{ lookup('env','DATA_PERSISTENCE') }}"
+    lib_image: "{{ lookup('env','LIB_IMAGE') }}"
+    liveness_label: "{{ lookup('env','LIVENESS_APP_LABEL') }}"
+    liveness_namespace: "{{ lookup('env','LIVENESS_APP_NAMESPACE') }}"
+    n_delay: "{{ lookup('env','NETWORK_DELAY') }}" 
+    openebs_ns: "{{ lookup('env','OPENEBS_NAMESPACE') }}"
+
+  vars_files:
+    - /mnt/parameters.yml
+    - /experiments/openebs/openebs_components.yml
+  
+  tasks:
+    - block:
+
+      - include_tasks: /utils/runtime/create_testname.yml  
+
+      - include_tasks: openebs_pool_network_delay_ansible_prerequisites.yml
+
+      # GENERATE EXPERIMENT RESULT NAME AND CREATE CHAOS RESULT RESOURCE
+      - block:
+          - name: Construct chaos result name (experiment_name)
+            set_fact:
+              c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+            when: lookup('env','CHAOSENGINE')
```

Comment:
From where you are taking `CHAOSENGINE` value it is not in Job spec I guess?

Reply:
Also, Can you change the variable name `c_experiment` to `c_result` as it has the value of chaos-result?
And you have already defined `c_experiment` in vars section.